docs/source/data.md

```diff
@@ -0,0 +1,36 @@
+# Data
+
+Without data, nothing can be done. To get you started, we recommend you download
+the [Spinal Cord MRI Public Database](https://openneuro.org/datasets/ds001919).
```

Comment:
We could also add a link to toy dataset (only a few images vs. 8 Go dataset). We could then use this small dataset in our tutorial code.

Reply:
Good point-- issue opened: https://github.com/neuropoly/ivado-medical-imaging/issues/253